### ISSUE TITLE
Feature Request: Ability to export DB (or caching of PWs in app)?

### ISSUE THREAD

**Issue by gregnorc** [SpiderOak/Encryptr#278]
First off, thanks for all your work on this - been using a while and liking it.

However, I do need an internet connection to access my PWs, not all of which are for apps that require and internet connection.

It'd be nice if there was a simple export function - maybe output a .CSV that could be stored in a safe place, like a Veracrypt volume.

Alternatively, if it's safe to do so, caching PWs from the last pull down would be nice. (User would still need to enter a passphrase to access). But I'm not a strong cryptographer, so I hesitate to suggest the latter without knowing the security implications.

Thanks for your help.

-Greg

**Comment by EMH333** [comment]
Yes, I agree this would be a nice feature to have. I use Encryptr often for remembering things (not necessarily passwords) and for the most part I am off line when I try and store information, until I remember I need to be online. If off line access was available, it would dramatically help me.

The only security concern I could see would be if someone got the database and then could attempt to crack it off line without need to go to the server. However, I am confident that it would take them too long for this to be a valid issue. It would also have to handle if someone were to add/edit a document while off line but you could simply restrict that.

-Ethan

**Comment by EarthLord** [comment]
Yes, this will be an amazing feature. Caching the encrypted database locally would help whenever there's on outage (which happened a day ago). Also whenever travelling without the Internet or travelling where there's a firewall. Thanks!

**Comment by hnykda** [comment]
:+1:

**Comment by alancliddell** [comment]
Yes, this would be nice to have. More generally import/export would help in situations where people use multiple password managers (e.g., [pass](https://www.passwordstore.org/) on the command line when a GUI is not available).

**Comment by franckl** [comment]
is this project still active ? no commit in 6 months and no way to export our passwords unfortunately

**Comment by rjgonza** [comment]
I doesn't look like it. I think a great alternative is KeePass and you own
cloud storage. I have switched. This was really nice, but this missing
feature is a deal breaker.

-
sent from mobile

On Feb 17, 2017 9:32 AM, "franckl" <notifications@github.com> wrote:

is this project still active ? no commit in 6 months and no way to export
our passwords unfortunately

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/SpiderOak/Encryptr/issues/278#issuecomment-280665064>,
or mute the thread
